store/tikv/region_request.go

```diff
@@ -94,8 +94,14 @@ func (s *RegionRequestSender) SendReqCtx(bo *Backoffer, req *tikvrpc.Request, re
 		}
 	})
 
+	var replicaRead kv.ReplicaReadType
```

Comment:
how about adding a metric for `ReplicaReadType` to let follower read time be observable in granfana

Reply:
Sure, let me add it.

Reply:
I was about to add a new label `replica` to those duration metrics. But I'm not sure what is the right convention to use. Can you give me some advice, thanks.

Reply:
Looks like many metrics are populated with specific label statically in code.  Adding a new label to it would make it harder to do so.